tests/vcr_interceptor_test.py

```diff
@@ -37,11 +38,106 @@ def test_request_cassette_with_two_requests():
 
 
 def test_too_many_requests():
-    @intercept_requests('cassette3', generate=False)
+    @intercept_requests('tests/cassette3', generate=False)
     def test_three_request_for_cassette_with_two_requests():
         urllib.request.urlopen('http://www.iana.org/domains/reserved').read()
         urllib.request.urlopen('http://www.iana.org/domains/reserved').read()
         urllib.request.urlopen('http://www.iana.org/domains/reserved').read()
 
     with pytest.raises(CannotOverwriteExistingCassetteException):
-        test_three_request_for_cassette_with_two_requests()
\ No newline at end of file
+        test_three_request_for_cassette_with_two_requests()
+
+
+def test_ignore_on_match():
+    """
+    Test 'ignore_on_match'.
+    """
+    @intercept_requests('tests/test_cassette', generate=True)
+    def actual_request():
+        requests.get('http://httpbin.org/anything',
+                     params={'param1': 'value1'})
```

Comment:
This test actually goes through to the internet, it doesn't rely on the response but I'm not sure if it would fail if the request times out for exmaple?

Reply:
Added timeout for API call.